signin.html

```diff
@@ -27,22 +27,22 @@
       <form class="sign">
         <div class="sign-text">
           <label for="email">이메일</label>
-          <input name="email" type="email">
+          <input id="sign-email" name="email" type="email">
+          <div id="email-message"><!-- 이메일 입력 정보 --></div>
         </div>
 
         <div class="sign-text">
           <label for="password">비밀번호</label>
           <div class="password-icon">
-            <input name="password" type="password">
-            <img class="icon-eye-off" src="images/eye-off.svg" alt="eye-off">
+            <input id="sign-password" name="password" type="password">
+            <img class="icon-eye" src="./images/eye-off.svg" alt="eye-off">
           </div>
+          <div id="password-message"><!-- 비밀번호 입력 정보 --></div>
         </div>
 
-        <div class="sign-button">
-          <div>로그인</div>
-        </div>
+        <button class="btn signin">로그인</button>
```

Comment:
```suggestion
        <button class="btn signin" type="submit">로그인</button>
```

form과 button type submit의 조합으로 더 강력하게 관리 할 수 있습니다

더 좋은 이유는 위의 코멘트에 작성해놨습니다!

그리고, form 과 submit button은 항상 커플처럼 붙혀 작성하는 편이 시맨틱하기도 해요!